src/map/storage.cpp

```diff
@@ -214,6 +214,9 @@ static enum e_storage_add storage_canAddItem(struct s_storage *stor, int idx, st
 
 	if (!stor->state.put)
 		return STORAGE_ADD_NOACCESS;
+	
+	if (items[idx].card[0] == CARD0_PET && items[idx].attribute == 1)
+		return STORAGE_ADD_INVALID;
```

Comment:
If you could move this up above the `STORAGE_ADD_NOACCESS` check I would appreciate it!